functional-tests/steps/then.js

```diff
@@ -10,14 +14,86 @@ Then(
                 value = "//DIV[@class='auth-footer']"
                 break;
             case "MAS":
-                value = "//DIV[@class='dashboard-heading'][text()='MAS']"
+                value = "//DIV[@class='dashboard-heading'][text()='Medicines Awareness Service']"
                 break;
             case "You have been signed out.":
-                value = "//DIV[@class='alert_1wamaxc-o_O-info_1lss0us']"
-                break;             
+                value = "//DIV[@class='css-mv6v0s']"
+                break; 
+            case "Your changes have been saved successfully":
+                value = "//DIV[@class='css-ctpeu']"
+                break;          
             default:
                 value = "div.dashboard-header";
           }
           checkContainsText("element", value, false, text);
     }
 );
+Then(
+    /^I expect to see a list of items$/, () => {
+        var table = ("table.Table.ItemList");
+        browser.isExisting(table);
+        browser.pause(2000);
+    }          
+);
+Then(
+    /^I expect a create a new item form pops up$/, () => {
+        var form = ("//DIV[@class='css-s2cbvv']");
+        browser.isExisting(form);
+        browser.getValue(form);
+    }          
+);
+Then(
+    /^I can add an Evidence type$/, () => { 
+        browser.click( "[for=evidenceType] .Select-control .Select-placeholder");    
+        browser.waitForVisible("#react-select-3--list", 5000);
+        browser.click("#react-select-3--option-16");
+        browser.pause(5000);
+    }          
+);
+Then(
+    /^I expect the new record is added to the list of items$/, () => {
+        var item = ("(//A[@class='ItemList__value ItemList__value--text ItemList__link--interior ItemList__link--padded ItemList__value--truncate'][text()='NSAIDs Automated test again'][text()='NSAIDs Automated test again'])[3]");
```

Comment:
I am not very familiar with XPATH selectors but this one is quite big and therefore is probably fragile. It is preferable to use CSS selectors too as these are more widely used/known.